examples/seismic/tti/operators.py

```diff
@@ -152,7 +152,8 @@ def ForwardOperator(model, source, receiver, time_order=2, space_order=4,
                            offset=model.nbpml)
     stencils += src.inject(field=v, u_t=ti + 1, expr=src * dt * dt / m,
                            offset=model.nbpml)
-    stencils += rec.interpolate(expr=u, u_t=ti, offset=model.nbpml)
-    stencils += rec.interpolate(expr=v, u_t=ti, offset=model.nbpml)
-
+    stencils += rec.interpolate(expr=u + v, u_t=ti, offset=model.nbpml)
```

Comment:
Supposed to be interpolate the sum, saw it so changed it here even so not really core of the PR.